src/components/Table/SortableTable.js

```diff
@@ -74,6 +74,7 @@ function defaultRenderRow(
 
 function getSelectableCell(row, rowSelected, onSelect) {
   const selectRowId = uniqueId('select-row-');
+  const disabled = Object.hasOwn(row, 'disabled') ? row.disabled : false;
```

Comment:
Does using `Object.hasOwn` fix an edge case? I can't think of how this would be different than just passing `row.disabled` to `<input>`.

Reply:
good points. I think there was something with `selectable` that it made sense here, but you're right with the disabled. 

I'll update that then merge :)